index.js

```diff
@@ -74,4 +75,43 @@ mOcKiNgCaSe.overrideString = () => {
   return mOcKiNgCaSe;
 }
 
+/**
+ * @param {string|string[]} input The user-given input.
+ * @return If the given input is an array of strings.
+ */
+function isArrayOfStrings(input) {
+  if (!Array.isArray(input)) {
+    return false;
+  }
+
+  if (input.some((value, i) => typeof value !== 'string')) {
+    throw TypeError(`Expected array of strings but got type '${typeof value}' at index ${i}`);
```

Comment:
[Array.prototype.some()
](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) returns `true` or `false`.
**Note:** `value` and `i` are not accessible inside the `TypeError` and instead of 
```js
TypeError: Expected array of strings but got type '${typeof value}' at index ${i}
```
the 
```js
ReferenceError: i is not defined
``` 
is thrown. So it's better to use old version of `isArrayOfStrings`